database/src/migrations/20240124141800_observation_scientific_name.ts

```diff
@@ -11,8 +12,34 @@ export async function up(knex: Knex): Promise<void> {
   await knex.raw(`--sql 
     SET search_path = 'biohub';
 
-    ALTER TABLE survey_observation ADD itis_tsn integer;
-    ALTER TABLE survey_observation ADD itis_scientific_name varchar(300);
+    ALTER TABLE survey_observation ADD COLUMN itis_tsn INTEGER;
+    ALTER TABLE survey_observation ADD COLUMN itis_scientific_name VARCHAR(300);
+    COMMENT ON COLUMN survey_observation.wldtaxonomic_units_id IS '(Deprecated) The species associated with the observation.';
+    COMMENT ON COLUMN survey_observation.itis_tsn IS 'The ITIS TSN identifier for the species associated with the observation.';
+    COMMENT ON COLUMN survey_observation.itis_scientific_name IS 'The scientific name for the species associated with the observation.';
+
+    ALTER TABLE study_species ADD COLUMN itis_tsn INTEGER;
+    ALTER TABLE study_species ALTER COLUMN wldtaxonomic_units_id DROP NOT NULL;
+    COMMENT ON COLUMN study_species.wldtaxonomic_units_id IS '(Deprecated) System generated UID for a taxon..';
+    COMMENT ON COLUMN study_species.itis_tsn IS 'The ITIS TSN identifier for the species associated with the survey.';
+
+    -- TODO Currently, we are using a placeholder ITIS species. Instead, we should map all known species used in prod from their elsaticsearch taxonomic ID to their ITIS TSN.
+
+    -- Placeholder:
+      UPDATE study_species SET itis_tsn = 202384;
+
+    -- Add not null constraint
+    ALTER TABLE study_species ALTER COLUMN itis_tsn SET NOT NULL;
```

Comment:
We may as well do the same thing for the itis_scientific_name (set a default placeholder and mark it not null)

Reply:
Good call

Reply:
Actually it seems like this table (`study_species`) doesn't have a column `itis_scientific_name`. Or were you referring to a different table?

Reply:
Right (remembering now). We only added scientific name to the other table to support sorting (and because it should be safe anyways). I think we opted not for this table in light of there being no strict requirement for it, yet.

Disregard my previous comment

Reply:
Can you add a todo item to this page, so we don't forget to patch the TSNs when we release: https://apps.nrs.gov.bc.ca/int/confluence/display/TASHIS/SIMS+Releases